packages/vite-app-ts/package.json

```diff
@@ -22,29 +25,31 @@
     "@babel/runtime": "^7.16.7",
     "@emotion/react": "^11.7.1",
     "@ethersproject/abi": "^5.5.0",
+    "@ethersproject/abstract-provider": "^5.5.1",
```

Comment:
Same as above, can't find use case (possibly just misunderstanding of how it works)

Reply:
i forgot the reason why i needed it.  but it doesn't hurt as vite will tree shake uneeded things